components/navigation/NavBar.js

```diff
@@ -69,7 +70,8 @@ export default function NavBar ({
 
           <div className="relative">
             <NavItem text="Community" onClick={() => showMenu('community')} hasDropdown />
-            {open === 'community' && <NavMenu items={communityItems} />}
+            {/* {open === 'community' && <NavMenu items={communityItems} />} */}
```

Comment:
If certain piece of code is not needed, you can remove it instead of commenting it. Please do remove this comment.

Reply:
Yikes, must have been an oversight, thanks. removed.